sdk/keyvault/keyvault-certificates/swagger/README.md

```diff
@@ -20,7 +20,7 @@ input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/7452e1c
 output-folder: ../
 source-code-folder-path: ./src/generated
 hide-clients: true
-package-version: 4.9.1
+package-version: 4.10.0-beta.1
```

Comment:
Now that keyvault is generated from TypeSpec and is stable, we should go back and clean up all the swagger remnants from the migration.

swagger/README.md can be deleted as well as any references to it (such as version constants in package.json etc)